app/services/gmail/delete.py

```
@@ -403,34 +370,37 @@ def delete_emails_bulk_background(senders: list[str]) -> None:
-            state.delete_bulk_status["progress"] = 40 + int(
-                (deleted / total_emails) * 60
-            )
-            state.delete_bulk_status["message"] = (
-                f"Deleted {deleted}/{total_emails} emails..."
+            progress = 40 + int((deleted / total_emails) * 60)
+            state.update_delete_bulk_status(
+                deleted_count=deleted,
+                progress=progress,
+                message=f"Deleted {deleted}/{total_emails} emails...",
             )
     except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)